readsbrrd.c

```diff
@@ -578,7 +578,7 @@ int main(int argc, char** argv) {
     // Run this until we get a termination signal.
     while (!readsbrrd_exit) {
         clock_gettime(CLOCK_REALTIME, &ts);
-        ts.tv_sec += (__time_t) (rrd.step * 1.5);
```

Comment:
Not a correct cast. See https://clang.llvm.org/extra/clang-tidy/checks/bugprone/misplaced-widening-cast.html

Change to

```
(time_t) rrd.step * 1.5;
```

Reply:
Thanks, fixed.